commands/FBDelay.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python
+import string
+import lldb
+import fblldbbase as fb
+import fblldbobjcruntimehelpers as runtimeHelpers
+from threading import Timer
+
+
+def lldbcommands():
+  return [
+    FBDelay()
+  ]
+
+class FBDelay(fb.FBCommand):
+  def name(self):
+    return 'zzz'
+
+  def description(self):
+    return 'Executes specified lldb command after delay.'
+
+  def options(self):
+    return []
+
+  def args(self):
+    return [
+      fb.FBCommandArgument(arg='delay in seconds', type='float', help='time to wait before executing specified command'),
+      fb.FBCommandArgument(arg='lldb command', type='string', help='another lldb command to execute after specified delay', default='process interrupt')
+    ]
+
+  def run(self, arguments, options):
+    lldb.debugger.SetAsync(True)
+    lldb.debugger.HandleCommand('continue')
+    delay = float(arguments[0])
+    command = str(arguments[1])
+    t = Timer(delay, lambda: self.runDelayed(command))
+    t.start()
+
+  def runDelayed(self, command):
+    lldb.debugger.HandleCommand('process interrupt')
+    lldb.debugger.HandleCommand(command)
```

Comment:
I actually set a default value on the command option, it probably got lost because I didn't break it at certain line length (is there a style guide you'd like to follow here?)
The default parameter is `, default='process interrupt'`
Should I add a check for an empty command and replace it with default value?